### PR TITLE
Branded base theme

### PR DESCRIPTION
**Pull request by matthieuh**
How to test this theme:

- `npm i`
- `pip3 install -r requirements.txt`
- `npm start`
- `mkdocs serve` (in an other session)
- point your browser to localhost:8000

=> product can be switched changing the product property in `mkdocs.yml`. Possible values are `proxy|mesh|enterprise|pilot`

=> I still need to check with @ldez which Pipy account we should use to publish this package.

### CODE REVIEW

**Review by paulocfjunior — CHANGES_REQUESTED** [review]
I just found a couple of issues when testing, they might not be tough:

- The Table of contents is showing on the footer:
Screen Shot 2021-01-04 at 08 44 41

- The dropdown indicator in the sidebar is rolling up, we can edit the transform so it keeps centralized when opening/closing:
https://user-images.githubusercontent.com/38889364/103539018-0ca24280-4e76-11eb-8688-4870c2ee998c.mp4



- The Open Positions card is not matching the [design](https://app.zeplin.io/project/5d25901c5e394f7a80f769a4/screen/5f5fae2f9508f77b16f3ba3d):
Screen Shot 2021-01-04 at 08 44 13

File: requirements.txt [hunk 1]
```diff
@@ -1,23 +1,3 @@
...
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
-# IN THE SOFTWARE.
-
 # Direct dependencies
 mkdocs>=1.1
 Pygments>=2.4
```
[the remark this answers is not in the document]
Comment (matthieuh) [reply requirements.txt]: Reading the mkdocs-material doc it has to be installed separately: https://github.com/squidfunk/mkdocs-material/blob/a339a8afb61fc7c7caf99561141d030dee42111d/docs/customization.md

My guess is that this is dev dependencies that should be bundled with the theme.

File: src/assets/stylesheets/main/layout/_nav.scss [hunk 2]
```diff
@@ -571,7 +571,7 @@
 
       // Rotate icon for expanded lists
       .md-nav__item--nested .md-nav__toggle:checked ~ .md-nav__link & {
-        transform: rotate(90deg);
+        transform: translate(0, 50%) rotate(90deg);
```
Comment (paulocfjunior) [inline src/assets/stylesheets/main/layout/_nav.scss:574]: Even though it's transforming to 50%, it doesn't seem to be centered yet, look:

Current behavior:

https://user-images.githubusercontent.com/38889364/103663523-f2d52e00-4f4f-11eb-8d52-716c69b77c4a.mp4

Expected:

https://user-images.githubusercontent.com/38889364/103663542-fa94d280-4f4f-11eb-9c7c-0acbc77a4afd.mp4

WDYT?

**Comment by paulocfjunior** [comment]
I think we should also update the social network links and github repo link:
Screen Shot 2021-01-05 at 12 21 56
Screen Shot 2021-01-05 at 12 21 47

**Comment by matthieuh** [comment]
> I think we should also update the social network links and github repo link:
> Screen Shot 2021-01-05 at 12 21 56
> Screen Shot 2021-01-05 at 12 21 47

It will use the right one when use in product docs based on the mkdocs configuration

File: src/assets/stylesheets/main/layout/_nav.scss (hunk 2 above)
Comment (matthieuh) [reply src/assets/stylesheets/main/layout/_nav.scss:574]: Seems a nice to have to me. Maybe we should put in an issue. Wdyt?

**Review by paulocfjunior — APPROVED** [review]

**Pull request merged**